book/website/requirements.in

```diff
@@ -7,3 +7,4 @@ matplotlib
 numpy
 pandas
 sphinx-panels
+tornado>=6.4.2 #temporary fix to https://github.com/the-turing-way/the-turing-way/security/dependabot/15
```

Comment:
I don't think we need the comment, this PR (or the commit message) will have the required metadata for the reason for the pin

Reply:
comment removed